patroni/utils.py

```diff
@@ -911,3 +912,35 @@ def enable_keepalive(sock: socket.socket, timeout: int, idle: int, cnt: Optional
 
     for opt in keepalive_socket_options(timeout, idle, cnt):
         sock.setsockopt(*opt)
+
+
+def shell_quote(opt: str) -> str:
+    """Quote a string intended to be used as part of a shell command.
+
+    Applies quoting on POSIX compliant systems only, on ``win32`` platforms
+    will just return the original string unquoted.
+
+    :param opt: String to be quoted
+    :returns: Quoted string
+    """
+    if sys.platform != 'win32' and requires_quoting(str(opt)):
+        return quote(str(opt))
```

Comment:
`opt` is already a string.
```suggestion
    if sys.platform != 'win32' and requires_quoting(opt):
        return quote(opt)
```

Reply:
Sorry, I should have removed that before pushing again.